### PR TITLE
Sign in

### PR DESCRIPTION
**Pull request by theVivek-kumar**
created signup page please review

### CODE REVIEW

File: SignIn/SignIn.css [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+    margin: 1rem;
+    justify-content: center;
+
+}
+
+.Login-input {
+    background-color: #3b82f6;
```
Comment (Adarsh-Patel-Dev) [inline SignIn/SignIn.css]: use color variables

File: SignIn/SignIn.css [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    display: flex;
+    justify-content: flex-start;
+    margin-top: 1rem;
+
+}
+
+.login-Fp-header {
+    display: flex;
+    justify-content: center;
+    color: #444444;
```
Comment (Adarsh-Patel-Dev) [inline SignIn/SignIn.css]: same here
use color var

File: SignIn/SignIn.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+}
+
+body {
+    background-color: #ebebeb;
+}
+
+:root {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    color: rgb(32, 32, 32);
```
Comment (Adarsh-Patel-Dev) [inline SignIn/SignIn.css]: use color variable

**Review by Adarsh-Patel-Dev — APPROVED** [review]

File: SignIn/SignIn.css [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+:root {
+        font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+        --main-heading: rgb(32, 32, 32);
+        --font-color: #282a36;
+        --BACKGROUND-color: #f39494;
+        --PRIMERY-color: rgb(12, 109, 165);
+        --SECONDERY-color: rgb(255, 196, 1);
+        --FOOTER-color: #202020;
+        --red-icon-color:red;
+        --white-text-color:white;
```
Comment (sharmayatendra) [inline SignIn/SignIn.css:22]: either use RGB or hex

File: SignIn/SignIn.css [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+.navbar-md-head {
+    display: flex;
+    justify-content: space-around;
+    background-color: #482880;
```
Comment (sharmayatendra) [inline SignIn/SignIn.css]: don't hard code color values

File: SignIn/SignIn.css [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+    justify-content: space-around;
+
+
+}
+
+.fa-search {
+    padding: 4px;
+    font-size: 1.6rem;
+    background-color: var(--white-text-color);
+    color: rgb(95, 95, 95);
```
Comment (sharmayatendra) [inline SignIn/SignIn.css]: use variable here as well

**Review by sharmayatendra — APPROVED** [review]
just made suggested changes else good to go

**Pull request merged**